modules/metering-overview.adoc

```diff
@@ -12,6 +12,45 @@ Metering focuses primarily on in-cluster metric data using Prometheus as a defau
 
 You can install metering on {product-title} 4.x clusters and above.
 
+[id="metering-overview-install-metering"]
+== Installing metering
+You can install metering using the CLI and the web console on {product-title} 4.x and above. To learn more see xref:../metering/metering-installing-metering.adoc#metering-install-prerequisites_installing-metering[installing metering].
+
+[id="metering-overview-upgrade-metering"]
+== Upgrading metering
+You can upgrade metering by updating the Metering Operator subscription. Review the following tasks:
+
+* The `MeteringConfig` custom resource specifies all xref:../metering/configuring_metering/metering-about-configuring.adoc#metering-about-configuring[the configuration details for your metering installation]. When you first install the metering stack, a default `MeteringConfig` custom resource is generated. Use the examples in the documentation to modify this default file.
+
+* xref:../metering/reports/metering-about-reports.adoc#metering-about-reports[A report custom resource] provides a method to manage periodic Extract Transform and Load (ETL) jobs using SQL queries. Reports are composed from other metering resources, such as `ReportQuery` resources that provide the actual SQL query to run, and `ReportDataSource` resources that define the data available to the `ReportQuery` and Report resources.
```

Comment:
s/\`ReportQuery\` and Report resources. / \`ReportQuery\` and \`Report\` resources.

Reply:
acked